### ISSUE TITLE
Selection

### ISSUE THREAD

**Issue by obriencole11** [BWoodward95/RigBox#7]
Anything that handles selection should be done at the highest level. This means any code below the surface should not query the users selection, instead pass nodes or data.

So instead of this:

```
class IKFK_System():

    def __init__(self):    
        self.selection = cmds.ls(sl=True)
```

Do this:

```
class IKFK_System():

    def __init__(self, selection):    
        self.selection = selection
```

**Comment by BWoodward95** [comment]
yeah, once this is connected to the rest of the autorigger, user-input selection will be replaced with variables created by createskeleton.py
